src/pages/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+function LoginPage(){
+
+function handleSubmit(){
+    console.log ("Hello world!")
+}
+    return(
+   
+    <div className='landingPage'>
+     
+     <h1>Login Page</h1>
+     <form className="formContainer">
+      <label htmlFor="email" >Email</label>
```

Comment:
ah - nice catch on using htmlFor instead of just 'for' here!

Reply:
that was Dasha's idea